neptune/new/internal/operation_processors/factory.py

```diff
@@ -38,6 +38,8 @@
 
 __all__ = ["get_operation_processor"]
 
+from ...sync.utils import create_dir_name
```

Comment:
please format this as a normal import (without leading ellipsis ;)) + put it with the others (before `__all__`)

Reply:
Sorry, I didn't notice this.

And definitely I should move `__all__` before imports.